src/main/java/edition/academy/seventh/display/BookFilter.java

```diff
@@ -0,0 +1,62 @@
+package edition.academy.seventh.display;
+
+/**
+ * Available {@link Pagination} book filters. Each {@code Enum constant} implements value
+ * initialized with native SQL query adequates to the name meaning.
+ *
+ * @author Kamil Rojek
+ */
+public enum BookFilter implements Filter {
+
+  /** Sorts in ascending manner books by ID. */
```

Comment:
Sort by ID in ascending order?

Reply:
Wydaje mi się, że opis jest ok bo sam filtr sortuje rosnąco po ID.